src/index.html

```diff
@@ -17,6 +17,22 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a href = "#" class="logo">
+        <img src="images/logo.png" alt="Moyo Logo">
+      </a>
+      <nav>
+        <ul>
+          <li><a href="#"class ="is-active">Apple</a></li>
+          <li><a href="#">Samsung</a></li>
+          <li><a href="#">Smartphones</a></li>
+          <li><a href="#"class ="hover"></a>Laptops & Computers</a></li>
```

Comment:
There is a misplaced closing tag here. The `</a>` tag should be placed after the text `Laptops & Computers`, not before it. Also, ensure there is a space between `href="#"` and `class="hover"`.